bindings/go/src/fdb/subspace/subspace.go

```diff
@@ -54,6 +54,10 @@ type Subspace interface {
 	// Subspace prepended.
 	Pack(t tuple.Tuple) fdb.Key
 
+	// PackWithVersionstamp is similar to Pack, but afford for an
+	// IncompleteVersionstamp in the tuple
+	PackWithVersionstamp(t tuple.Tuple) (fdb.Key, error)
```

Comment:
Check out the documentation for the Python version of this function for a descriptive blurb you can roughly copy here: https://apple.github.io/foundationdb/api-python.html

Reply:
@ajbeamon - roughly copied.